index.html

```diff
@@ -37,8 +37,8 @@
 
     <section class="wrapper">  
       <div class="introduction">
-        <h1>I’m Alexandra<br>Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website 
+        <h1 class="name__int">I’m Alexandra<br>Glad to see you!</h1>
```

Comment:
- [optional] please, the side content of your cards should cover at least 3/4 of the card, so it matches the Figma design well. You can bring down the button a little
- [optional] please kindly add the hover effects to your buttons. 
![Screenshot (853)_LI](https://user-images.githubusercontent.com/65298048/222854384-7c709905-d13f-40d4-99f0-f87a8a86baf0.jpg)

Reply:
Thank your kindly support @tufoinnkuo10